OgreMain/src/OgreAnimationState.cpp

```diff
@@ -350,26 +347,22 @@ namespace Ogre
         // lock source
             OGRE_LOCK_AUTO_MUTEX;
 
-        AnimationStateMap::iterator i, iend;
-        iend = target->mAnimationStates.end();
-        for (i = target->mAnimationStates.begin(); i != iend; ++i) {
-            AnimationStateMap::const_iterator iother = mAnimationStates.find(i->first);
+        for (auto& s : target->mAnimationStates) {
+            AnimationStateMap::const_iterator iother = mAnimationStates.find(s.first);
             if (iother == mAnimationStates.end()) {
-                OGRE_EXCEPT(Exception::ERR_ITEM_NOT_FOUND, "No animation entry found named " + i->first, 
-                    "AnimationStateSet::copyMatchingState");
+                OGRE_EXCEPT(Exception::ERR_ITEM_NOT_FOUND, "No animation entry found named " + s.first, \
```

Comment:
```suggestion
                OGRE_EXCEPT(Exception::ERR_ITEM_NOT_FOUND, "No animation entry found named " + s.first,
```
the issue is that the line ends with a whitespace